pkg/pss/evaluate.go

```diff
@@ -71,7 +79,51 @@ func exemptKyvernoExclusion(defaultCheckResults, excludeCheckResults []pssutils.
 	for _, excludeResult := range excludeCheckResults {
 		for _, checkID := range pssutils.PSS_controls_to_check_id[exclude.ControlName] {
 			if excludeResult.ID == checkID {
-				delete(defaultCheckResultsMap, checkID)
+				for _, excludeFieldErr := range *excludeResult.CheckResult.ErrList {
+					excludeField, _, _, _ := parseField(excludeFieldErr.Field)
```

Comment:
If only the first field returned is needed, I think that maybe we can optimize the `parseField` function to avoid it doing extra logic.

Reply:
`parseField` is needed to make full computation in the `exemptContainerLevelExclusion` function. All its results would be needed.

Reply:
I mean that for `exemptPodLevelExclusion`, it only needs the first field returned by `parseField`, which leads to extra logic handled by `parseField`, and I think we should avoid that part of it?